Code/GraphMol/ChemTransforms/MolFragmenter.cpp

```diff
@@ -591,6 +594,227 @@ ROMol *fragmentOnBRICSBonds(const ROMol &mol) {
   }
   return fragmentOnBonds(mol, bondPatterns, &(atomEnvs.get()));
 }
+} // End of MolFragmenter
 
-}  // end of namespace MolFragmenter
+namespace {
+// Get the atom label - this might be useful as a util class
+unsigned int get_label(const Atom *a, const MolzipParams &p) {
```

Comment:
need a PRECONDITION on the atom pointer

Reply:
I'll add it but I don't really think the atoms can be null here, but safety first.